### DISCUSSION TITLE
十一月更新 - 1

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#591] [通知]
## 🚀 功能

- 添加分集索引视图 @Richasy (#589)

## 🐛 Bug 修复

- 修复字幕在小窗模式下不显示的问题 @Richasy (#588)
- 修复番剧不加载字幕的问题 @hez2010 (#576)
- 更改系统字体列表的读取方式以避免崩溃 @JasonWei512 (#554)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2111.1.0'>十一月更新 - 1</a>.</em>

**Comment by Jarry-YE** [comment]
大佬，可以加一个循环播放的按钮不？
